origami.json

```diff
@@ -5,53 +5,80 @@
     "support": "https://github.com/Financial-Times/o-forms/issues",
     "supportStatus": "active",
     "ci": {
-        "travis": "https://api.travis-ci.org/repos/Financial-Times/o-forms/builds.json"
+        "circle": "https://circleci.com/api/v1/project/Financial-Times/o-forms"
     },
     "demos": [
         {
-            "path": "/demos/text-inputs.html",
-            "expanded": true,
-            "description": ""
+            "name": "real-life",
+            "template": "demos/src/real-life.mustache",
+            "bodyClasses": "o-colors-page-background demo-real-life",
+            "dependencies": [
+                "o-colors@^2.4.7",
+                "o-fonts@^1.4.0",
+                "o-tabs@^1.4.2",
+                "o-buttons@^2.0.4",
+                "o-footer@^4.0.0",
+                "o-grid@^4.0.0"
+            ],
+            "expanded": false,
```

Comment:
It might be better to leave this one as `"expanded": true` and the rest of the demos as `false` I think this one is probably the most useful demo for people to see.